styles/ice.json

```diff
@@ -3,56 +3,221 @@
+					"name": "Vertical hard ink to white"
+				},
+				{
+					"slug": "gradient-11",
+					"gradient": "linear-gradient(to bottom, #37505d 50%, #466577 50%)",
+					"name": "Vertical hard ocean to slate"
+				},
+				{
+					"slug": "gradient-12",
+					"gradient": "linear-gradient(to bottom, #24343d 50%, #37505d 50%)",
+					"name": "Vertical hard ink to ocean"
+				}
+			],
+			"palette": [
+				{
+					"color": "#FFFFFF",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#cbd9e1",
+					"name": "Base / Two",
+					"slug": "base-2"
+				},
+				{
+					"color": "#ecf1f4",
+					"name": "Base / Three",
+					"slug": "base-3"
+				},
+				{
+					"color": "#24343d",
+					"name": "Contrast",
+					"slug": "contrast"
+				},
+				{
+					"color": "#37505d",
+					"name": "Contrast / Two",
+					"slug": "contrast-2"
+				},
+				{
+					"color": "#466577",
+					"name": "Contrast / Three",
+					"slug": "contrast-3"
+				}
+			]
+		},
 		"typography": {
 			"fontFamilies": [
 				{
-					"fontFamily": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
-					"name": "System Font",
+					"fontFace": [
+						{
+							"fontFamily": "Inter",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "300 900",
+							"src": [
+								"file:./assets/fonts/inter/Inter-VariableFont_slnt,wght.woff2"
+							]
+						}
+					],
+					"fontFamily": "\"Inter\", sans-serif",
+					"name": "Inter",
 					"slug": "heading"
 				},
 				{
-					"fontFamily": "\"Inter\", sans-serif",
-					"name": "Inter",
+					"fontFace": [
+						{
+							"fontFamily": "Jost",
+							"fontStyle": "normal",
+							"fontWeight": "100 900",
+							"src": ["file:./assets/fonts/jost/Jost-VariableFont_wght.woff2"]
+						},
+						{
+							"fontFamily": "Jost",
+							"fontStyle": "italic",
+							"fontWeight": "100 900",
+							"src": [
+								"file:./assets/fonts/jost/Jost-Italic-VariableFont_wght.woff2"
+							]
+						}
+					],
+					"fontFamily": "\"Jost\", sans-serif",
+					"name": "Jost",
 					"slug": "body"
 				}
+			],
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Small",
+					"size": "1rem",
+					"slug": "small"
+				},
+				{
+					"fluid": false,
+					"name": "Medium",
+					"size": "1.2rem",
+					"slug": "medium"
+				},
+				{
+					"fluid": {
+						"min": "1.5rem",
+						"max": "2rem"
+					},
+					"name": "Large",
+					"size": "2rem",
+					"slug": "large"
+				},
+				{
+					"fluid": {
+						"min": "2rem",
+						"max": "2.65rem"
+					},
+					"name": "Extra Large",
+					"size": "2.65rem",
+					"slug": "x-large"
+				},
+				{
+					"fluid": {
+						"min": "2.65rem",
+						"max": "3.5rem"
+					},
+					"name": "Extra Extra Large",
+					"size": "3.5rem",
+					"slug": "xx-large"
+				}
 			]
+		},
+		"layout": {
+			"contentSize": "620px",
+			"wideSize": "99vw"
```

Comment:
this makes wide width content virtually full width. I suppose that's the intent, just pointing it out

Reply:
The home looks a little weird on really big screens

Reply:
<img width="2372" alt="Screenshot 2023-10-16 at 12 05 53" src="https://github.com/WordPress/twentytwentyfour/assets/3593343/9221b11d-a4f9-46ac-8153-c3e90d6bf2fc">